src/platform/CYW30739/KeyValueStoreManagerImpl.cpp

```diff
@@ -128,6 +128,20 @@ CHIP_ERROR KeyValueStoreManagerImpl::_Delete(const char * key)
     return err;
 }
 
+CHIP_ERROR KeyValueStoreManagerImpl::EraseAll(void)
+{
+    wiced_result_t result;
+    wiced_hal_delete_nvram(PLATFORM_NVRAM_ID_ENTRY_INFO_STRING, &result);
+    if (result == WICED_SUCCESS)
+    {
+        return CHIP_NO_ERROR;
+    }
+    else
```

Comment:
nit: prefer no else after return.

maybe even



```cpp

return (result == WICED_SUCCESS) ? CHIP_NO_ERROR : CHIP_ERROR_PERSISTED_STORAGE_FAILED

```



you may want to actually print out the result in case of failure though as you lose the actual result error code here and return a generic `failed` status which is hard to debug.

Then again, logs seem disabled for this platform :/